README.md

```
@@ -1,90 +1,187 @@
 
-- Setup the backend application. You can find the instructions [here](https://github.com/glific/glific).
-- SSL is mandatory for both frontend and backend in development. You can find the instructions to generate ssl certificate [here](https://github.com/glific/glific#use-ssl-for-frontend-and-backend)
-- Configure below host in your `/etc/hosts`
-  `127.0.0.1 glific.test`
+## 📋 Table of Contents
+- [Pre-requisite](#-pre-requisite)
+  - [Software Dependencies](#software-dependencies)
+    - [npm (via asdf)](#npm-via-asdf)
+    - [yarn (via npm)](#yarn-via-npm)
+  - [Glific Backend](#glific-backend)
+- [Clone Frontend Repo](#-clone-frontend-repo)
```

markdownlint-cli2 (0.17.2)

Link fragments should be valid
null

(MD051, link-fragments)